packages/manager/apps/web-ongoing-operations/src/data/api/web-ongoing-operations.ts

```diff
@@ -1,48 +1,47 @@
 import { fetchIcebergV6, apiClient } from '@ovh-ux/manager-core-api';
-
-
-export type GetmeTaskDomainListParams = {
-  /** Filter the value of domain property (like) */
-  'domain': any;
-  /** Filter the value of function property (like) */
-  'function': any;
-  /** Filter the value of status property (&#x3D;) */
-  'status': any;
-};
+import { taskMeDomain } from '@/constants';
 
 export const getmeTaskDomainListQueryKey = ['get/me/task/domain'];
 
 /**
  * Get information about domain related tasks : List of domain tasks
  */
-export const getmeTaskDomainList = async (params: GetmeTaskDomainListParams): Promise<any> =>
-  apiClient.v6.get('/me/task/domain', { data: params });
+export const getmeTaskDomainList = async (): Promise<any> =>
+  apiClient.v6.get(taskMeDomain);
 
 export type GetmeTaskDomainIdParams = {
   /** Id */
-  'id'?: any;
+  id?: any;
 };
 
-export const getmeTaskDomainIdQueryKey = (params: GetmeTaskDomainIdParams) => [`get/me/task/domain/${params.id}`];
+export const getmeTaskDomainIdQueryKey = (params: GetmeTaskDomainIdParams) => [
+  `get/me/task/domain/${params.id}`,
```

Comment:
```suggestion
  ['get', 'me', 'task', 'domain', params.id],
```
It's better as an array rather than a string to revoke the cache